mcp/include/mcp.h

```diff
@@ -53,7 +53,7 @@ void force_incharge(void);
 // telemetry defines
 
 // how many options we have
-#define NUM_TELEMETRIES 4
+#define NUM_TELEMETRIES 6
```

Comment:
This is temporary right? highrate and all that stuff are replaced by EVTM

Reply:
specifically highrate and bio are replaced by evtmtdrss and evtmlos

Reply:
Yes, temporary! Once we complete end-to-end testing, we can remove the old implementations.

Reply:
Yup, TBD on what the flight config will look like...